Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:6.11-alpine
+RUN mkdir -p /code
+WORKDIR /code
+COPY package.json .
```

Comment:
Check if we can skip copying package.json by adding the root folder before yarning.